neuroscout_cli/commands/install.py

```diff
@@ -0,0 +1,72 @@
+from neuroscout_cli.commands.base import Command
+
+from datalad import api as dl
+from datalad.auto import AutomagicIO
+from os.path import isdir, exists, join
+import requests
+import json
+import tarfile
+
+
+class Install(Command):
+
+    ''' Command for retrieving neuroscout bundles and their corresponding
+    data. '''
+
+    def is_bundle_local(self):
+        local = isdir(self.bundle_name)
+        local &= exists(join(self.bundle_id, 'resources.json'))
+        local &= exists(join(self.bundle_id, 'events.tsv'))
+        local &= exists(join(self.bundle_id, 'analysis.json'))
+        return local
+
+    def download_bundle(self):
+        if not self.is_bundle_local():
+            endpoint = 'http://146.6.123.97/api/analyses/%s/bundle' % self.bundle_id
```

Comment:
This can go later, but I would add a config.py file or, env variables that configure things such as the remote neuroscout server address